### PR TITLE
Updated deps. Changed DClient constructor

### PR DESCRIPTION
**Pull request by mjou-ut**

### CODE REVIEW

File: src/stature/metrics.clj [hunk 1]
```diff
@@ -2,14 +2,19 @@
...
 (defrecord MetricsComponent [host port prefix client]
   component/Lifecycle
   (start [c]
-    (let [client (NonBlockingStatsDClient. prefix host port)]
+    (let [builder (.. (NonBlockingStatsDClientBuilder.)
+                      (prefix prefix)
+                      (hostname host)
+                      (port port)
+                      (resolve))
+          client (NonBlockingStatsDClient. builder)]
```
Comment (AfonsoTsukamoto) [inline src/stature/metrics.clj]: Have you tried type-hinting the builder? The class has 2 constructors so type-hinting might force it to use the one that takes a builder

Comment (mjou-ut) [reply src/stature/metrics.clj]: Yes

**Review by lukaszkorecki — APPROVED** [review]

**Comment by lukaszkorecki** [comment]
@mjou-uz I've released the snapshot version based on this branch 👍

**Comment by mjou-ut** [comment]
> @mjou-uz I've released the snapshot version based on this branch 👍

Cool, now I can deploy _dependants_

**Pull request merged**